main.py

```diff
@@ -710,6 +711,20 @@ def handle_link_up(self, event):
                 with evc.lock:
                     evc.handle_link_up(event.content["link"])
 
+    @listen_to("kytos/topology.updated")
+    def on_topology_update(self, event):
+        """Capture topology update event"""
+        self.handle_topology_update(event)
+
+    def handle_topology_update(self, event):
+        """Handle topology update"""
```

Comment:
Before processing a `kytos/topology.update` since this can get out of order. You'll need a single lock keeping track of `event.timestamp`, similar to what we have on pathfinder https://github.com/kytos-ng/pathfinder/blob/master/main.py#L155-L163, in the future we'll be able to leverage https://github.com/kytos-ng/kytos/issues/397 to facilitate, but for now this is needed.

Reply:
Added `lock`

Reply:
Added the timestamp check.